Sample-Notebooks/Example - Azure Storage VT Hash Lookup.ipynb

```diff
@@ -0,0 +1,196 @@
+{
```

Comment:
<p>It might be nice to make this a msticpy builtin query</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>